getnacelle/nacelle-js
# chore (root): add gatsby-source-filesystem to root dev dependencies

## WHY are these changes introduced?

Story [LS-1507](https://nacelle.atlassian.net/browse/LS-1507) 

## WHAT is this pull request doing?

Adds `gatsby-source-filesystem` to the root `package.json`'s dev dependencies. This allows us to build the `gatsby-starter` locally since `lerna`'s symlinks to internal packages breaks the node_modules resolution of `gatsby-source-filesystem` since it's a peer dependency of `gatsby-source-nacelle`.

## How to Test

1. Run `npm i`
2. Run `npm run bootstrap`
3. Run `npm run test:ci` to ensure tests run with no errors.
4. Run `npm run build` to ensure packages build with no errors.
5. `cd starters/gatsby`
6. Run `npm run build` to make sure the starter builds with all of the image nodes
